src/bpftrace.cpp

```diff
@@ -288,6 +291,15 @@ int BPFtrace::add_probe(ast::Probe &p)
       probe.mode = attach_point->mode;
       probe.async = attach_point->async;
 
+      if (has_iter)
+      {
+        LOG(ERROR) << "Only one iter probe allowed";
+        return 1;
```

Comment:
And also I don't think codegen_llvm's call to `add_probe()` checks for errors. We should probably make it check and bail on error

Reply:
right, I'll check on that